### PR TITLE
Fix/bindings get account assets

### PR DESCRIPTION
**Pull request by l4l**
### Description of the Change

Binding tests fixes for GetAccountAssets query that was missed in #1402 

### Benefits

Tests passes

### Possible Drawbacks 

None

### Running Tests

`cmake .. -DSWIG_JAVA=ON -DSWIG_PYTHON=ON -DSWIG_NODE=ON`
`make`
`ctest -R (python|java)`

### CODE REVIEW

File: shared_model/packages/javascript/tests/queryBuilder.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const accountId = 'admin@test'
 const assetId = 'coin#test'
 
 test('ModelQueryBuilder tests', function (t) {
-  t.plan(50)
+  t.plan(47)
```
Comment (nickaleks) [inline shared_model/packages/javascript/tests/queryBuilder.js]: What does this line mean?

Comment (l4l) [reply shared_model/packages/javascript/tests/queryBuilder.js]: number of tests :\

**Comment by lebdron** [comment]
Well, GetAccountAssets tests work in docker develop image after some actions, but other tests fail.
```
  ..................................................x............................................................................terminate called after throwing an instance of 'std:
:out_of_range'
  what():  bitset::set: __position (which is 18446744072523542336) >= _Nb (which is 43)
Aborted (core dumped)



  ---
    operator: throws
    expected: '/Hash set should contain at least one hash/'
    actual:   undefined
    at: Test.<anonymous> (/opt/iroha/iroha/shared_model/packages/javascript/tests/queryBuilder.js:92:5)
    stack: |-
  ...



  24 tests
  126 passed
  1 failed
```

**Comment by l4l** [comment]
javascripts are indeed failing, but that behavior was before the pr and requires shared model permissions (and maybe smth else)

**Pull request merged**